README.md

```diff
@@ -37,7 +37,7 @@ You are free to use any /16 to /28 CIDR block in the RFC 1918 private address ra
 
 ## VPC Template Parameters
 
-To deploy this VPC template, you'll need to know the VPC CIDR block, the three public, and three private subnet CIDR blocks. You will also choose whether the VPC will support highly available NAT Gateways, or, by default, a more cost effective single NAT Gateway.
+To deploy this VPC template, you'll need to know the VPC CIDR block, the three public, and three private subnet CIDR blocks. You will also choose whether the VPC will support highly available NAT Gateways and VPC Flow Logs, or, by default, a more cost effective single NAT Gateway without VPC Flow Logs enabled.
```

Comment:
Can we edit this paragraph to more clearly indicate that the choices of highly available NAT and the choice to include flow logs are independent choices from one other? I can see where, reading this, someone could assume that the highly available NAT choice also controls the flow logs choice. (The table below helps, to be sure)

Reply:
Just updated in the latest commit